README.md

```diff
@@ -98,25 +100,28 @@ progress".
 
 ## Internet-Draft Names
 
-Internet-Draft names appear inside the I-D, and are used as components of
-filenames for the various formats the I-D may appear in. The characters
-that may appear in an I-D name are restricted:
+The characters that may appear in an I-D name is restricted:
 
 * [ ] Ensure that the I-D name consists only of
   * the lowercase letters `a-z`,
   * the digits `0-9`,
   * or hyphens (`-`).
 
-The name of an I-D consists of several components, separated by a hyphen.
-No empty components are allowed. That is, consecutive hyphens may not appear
-in an I-D name.
+Internet-Draft names have one of the following two formats:
+
+    draft-{SOURCE}-{WGNAME}-{SHORT-DOC-TITLE}-{VV}
+
+    draft-{AUTHOR-NAME}-{SHORT-DOC-TITLE}-{VV}
```

Comment:
```suggestion
    `draft-{SOURCE}-{WGNAME}-{SHORT-DOC-TITLE}-{VV}`

    `draft-{AUTHOR-NAME}-{SHORT-DOC-TITLE}-{VV}`
```

Reply:
Why do you want this? Indented lines are taken as code-like markup; see https://github.com/richsalz/id-guidelines/tree/wordsmith-1

Reply:
ping @larseggert to know why he prefers backtick over indented lines.

Reply:
You're right - please disregard.